wings/1.0/configuration.md

```diff
@@ -29,3 +30,70 @@ docker:
       username: "registryusername"
       password: "registrypassword"
 ```
+
+## Custom Network Interfaces
+
+You can change the network interface that Wings uses for all containers by editing the network name; it is by default set to `pterodactyl_nw`. For example, to enable Docker host mode change the network name to `host`.
+
+::: warning
+Changing network mode to `host` grants Pterodactyl direct access to all machine interfaces and panel users can bind to any IP or Port even if it's not allocated to their container. You will lose all benefits of Docker network isolation. It is not recommended for public installations that are hosting other users' servers.
+:::
+
+### Example of usage
+
+```yml
+docker:
+  network:
+    name: host
+    network_mode: host
```

Comment:
This line doesn't do anything.

Reply:
It doesn't affect the network interface, but I've included it since it tends to confuse users when they're instructed to rename it. I guess it can be removed completely.